tools/scripts-core/scripts/build-storybook.js

```diff
@@ -1,14 +1,12 @@
 /* eslint-disable import/extensions */
 import spawn from 'cross-spawn';
 import { resolveBin } from '../utils/path-resolver.js';
-import { getPreset } from '../utils/preset.js';
+import { getStorybookConfiguration } from './start-storybook.js';
```

Comment:
I think getStorybookConfiguration should be in utils to not have dep between two script.